docs/DeveloperGuide.md

```diff
@@ -157,183 +198,49 @@ The `Storage` component,
-
-<img src="images/CommitActivityDiagram.png" width="250" />
-
-#### Design considerations:
-
-**Aspect: How undo & redo executes:**
-
-* **Alternative 1 (current choice):** Saves the entire scheduler.
-  * Pros: Easy to implement.
-  * Cons: May have performance issues in terms of memory usage.
-
-* **Alternative 2:** Individual command knows how to undo/redo by
-  itself.
-  * Pros: Will use less memory (e.g. for `delete`, just save the event being deleted).
-  * Cons: We must ensure that the implementation of each individual command are correct.
-
-_{more aspects and alternatives to be added}_
-
-### \[Proposed\] Data archiving
-
-_{Explain here how the data archiving feature will be implemented}_
-
-### \[Proposed\] Calendar view feature
-
-#### Proposed Implementation
-
-The proposed calendar view user interface implements the following operations:
-
-1. Provides an overview of all saved events in the scheduler to the user
-2. onClick() of calendar date (boxes) - List all the saved events the user have for that day
-
-The feature is going to be implemented by
-1. Creating the GUI of a calendar via JavaFX (Calendar.fxml and CalendarBox.fxml)
-2. When the program starts, it first fetches the existing events
-3. The calendar will next be "drawn" with the events for the current month
-4. The calendar includes button that allow users to go to either the next or previous month, each time the button is called, the calendar is "redrawn" with that particular month's events
-5. When a user were to click a date in the calendar (CalendarBox.fxml), it will list all existing events of that day to the user
-6. When a user were to add, edit or update an event, the calendar will be updated as well
-
-#### Design considerations:
-* **Alternative 1 (current choice):** Update the Calendar GUI whenever add/edit/delete an event.
-    * Pros: Easy to implement.
-    * Cons: May have performance issues in terms of memory usage.
-
-* **Alternative 2:** Update the Calendar GUI on if the month of the event is what the Calendar is current showing
-  itself.
-    * Pros: Will have better performance (e.g. for `add event`, no need to update the GUI if it is not what I am currently showing, if I were to click next).
-    * Cons: Implementation will become much more complex.
-
-#### Additional Feature
-1. onDoubleClick() of a calendar date (boxes) - Shows a popup GUI of user schedule, similar to that of a timetable for either that day or week
+<h3 id="help-command">Help Command</h3>
 
+<h3 id="exit-command">Exit Command</h3>
 
+<h2 id="documentation">Documentation, Logging, Testing, Configuration, Dev-Ops</h2>
 --------------------------------------------------------------------------------------------------------------------
-
-## **Documentation, logging, testing, configuration, dev-ops**
 * [Documentation guide](Documentation.md)
 * [Testing guide](Testing.md)
 * [Logging guide](Logging.md)
 * [Configuration guide](Configuration.md)
 * [DevOps guide](DevOps.md)
```

Comment:
This looks like its referencing to different files. Will this be an issue when we try to convert the DG to PDF?

Reply:
This was copied over from AB3. If there is an issue when converting to PDF, perhaps we could simply copy and paste over?